library-front/src/App.tsx

```diff
@@ -23,11 +24,23 @@ interface ContextSearch {
   setSearch: React.Dispatch<React.SetStateAction<string>>
 }
 
-const queryClient = new QueryClient()
+interface ContextFilters {
+  filters: BookFilter[]
+  setFilters: React.Dispatch<React.SetStateAction<BookFilter[]>>
+}
+
+interface ContextSort {
+  sort: (string | undefined)[] | undefined
+  setSort: React.Dispatch<React.SetStateAction<string[]>>
+}
+
+const queryClient = new QueryClient({ defaultOptions: { queries: { staleTime: 1000 * 60 * 5 } } })
 
 function App() {
   const [jwtToken, setJwtToken] = useState<Jwt | null>(null)
   const [search, setSearch] = useState<string>('')
+  const [filters, setFilters] = useState<BookFilter[]>([])
+  const [sort, setSort] = useState<(string | undefined)[] | undefined>()
```

Comment:
array of undefined values?

Reply:
The React Select component requested this type of data, even though the values will never be undefined, as the selectable options are always defined